NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs

```diff
@@ -306,6 +306,7 @@ public async Task PresentAsync(string? url = null)
         //Validated from startup
         if (!string.IsNullOrEmpty(url))
         {
+            _urlRow.SetText(url);
             await SearchUrlAsync(url);
```

Comment:
So it will paste the link and automatically start validation, correct? Maybe just pasting is enough?

Reply:
Yes it will automatically paste and start validation....if the user clicks the button to start a media again, then i'm 99% sure they'll press the validation button to configure the download.

It's different than reading from clipboard because the user might have had an old url or something in the clipboard that is different than what they'd like to download. This is more likely to occur hence a clipboard link isn't automatically validated.